### PR TITLE
[ko] Translate json with some validation

### PR DESCRIPTION
**Pull request by SilNex**

### CODE REVIEW

File: src/ko/validation-inline.php [hunk 1]
```diff
@@ -90,7 +90,7 @@
         'string'  => '이 문자열의 길이는 최소한 :min자이어야 합니다.',
         'array'   => '이 배열의 항목 수는 최소한 :min개의 항목이 있어야 합니다.',
     ],
-    'multiple_of'          => 'The value must be a multiple of :value',
+    'multiple_of'          => ':attribute 는 :value 의 배수여야 합니다.',
```
Comment (caouecs) [inline src/ko/validation-inline.php:93]: No :attribute here

**Review by caouecs — CHANGES_REQUESTED** [review]
Thank you, just a typo

**Comment by SilNex** [comment]
fixit !

**Comment by caouecs** [comment]
Thank you

**Pull request merged**